files/en-us/web/css/height/index.md

```diff
@@ -59,10 +59,6 @@ height: unset;
   - : Use the available space, but not more than [max-content](/en-US/docs/Web/CSS/max-content), i.e `min(max-content, max(min-content, stretch))`.
 - `fit-content({{cssxref("&lt;length-percentage&gt;")}})`
   - : Uses the fit-content formula with the available space replaced by the specified argument, i.e. `min(max-content, max(min-content, <length-percentage>))`.
-- {{cssxref("clamp", "clamp()")}}
```

Comment:
we only include Formal Syntax values in the values section, not functions that are part of those, so removed all the occurrences of functions within values, which included this clamp. added minmax above to make up for it.

Reply:
Right, ok. That's not even part of the work I am doing, but whatever.